### PR TITLE
Remove usage of deprecated Twig stuff

### PR DESCRIPTION
**Pull request by Koc**

### CODE REVIEW

File: Resources/views/CRUD/base_list.html.twig [hunk 1]
```diff
@@ -206,7 +206,7 @@ file that was distributed with this source code.
                         <div class="filter_container {{ admin.datagrid.hasActiveFilters ? 'active' : 'inactive' }}">
                             {% for filter in admin.datagrid.filters %}
                                 <div class="form-group">
-                                    {% if filter.label is not sameas(false) %}
+                                    {% if filter.label is not same as(false) %}
```
Comment (Soullivaneuh) [inline Resources/views/CRUD/base_list.html.twig]: `same as` fixes should be on another PR for history clarification.

Comment (Koc) [inline Resources/views/CRUD/base_list.html.twig]: why? it also deprecates

Comment (Soullivaneuh) [inline Resources/views/CRUD/base_list.html.twig]: Indeed, didn't noticed. Forget it. :-)

**Comment by Soullivaneuh** [comment]
Thank you @Koc.
